src/TimePicker/TimePicker.js

```diff
@@ -7,6 +7,39 @@ const CLOCK = ['am', 'pm'];
 const INVALID = 'is-invalid';
 const VALID = 'fd-input';
 class TimePickerItem extends Component {
+    static propTypes = {
+        buttonID: PropTypes.string,
+        buttonProps: PropTypes.object,
+        disabled: PropTypes.bool,
+        format12Hours: PropTypes.bool,
+        id: PropTypes.string,
+        inputId: PropTypes.string,
+        inputProps: PropTypes.object,
+        isValid: PropTypes.bool,
+        length: PropTypes.number,
+        placeholder: PropTypes.string,
+        showHour: PropTypes.bool,
+        showMinute: PropTypes.bool,
+        showSecond: PropTypes.bool,
+        style: PropTypes.string,
+        updateValue: PropTypes.func,
+        value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+        onChange: PropTypes.func
+    };
+    static defaultProps = {
+        value: null,
+        inputId: '',
+        buttonID: '',
+        style: '',
+        isValid: false,
+        length: 0,
+        disabled: false,
+        format12Hours: false,
+        showHour: false,
+        showMinute: false,
+        showSecond: false
+    };
+
```

Comment:
I believe these were moved to a static reference outside the class to match all other components.